app/Models/ServiceApplication.php

```
@@ -189,65 +189,51 @@ public function isBackupSolutionAvailable()
-                    ]);
+            // Parse the Docker Compose to find SERVICE_URL/SERVICE_FQDN variables DIRECTLY DECLARED
+            // for this specific service container (not just referenced from other containers)
+            $dockerComposeRaw = data_get($this->service, 'docker_compose_raw');
+            if (! $dockerComposeRaw) {
+                // Fall back to service-level port if no compose file
+                return $this->service->getRequiredPort();
+            }
 
-                    continue;
-                }
+            $dockerCompose = \Symfony\Component\Yaml\Yaml::parse($dockerComposeRaw);
```

PHPMD (2.15.0)

Avoid using static access to class '\Symfony\Component\Yaml\Yaml' in method 'getRequiredPort'. (undefined)

(StaticAccess)